### PR TITLE
Fix update role, add Molecule testing

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,10 +8,8 @@ python:
 
 jobs:
   include:
-  - name: "Testing ansible playbook `deploy-discourse.yml`"
-    env: TEST_ROLE=discourse
-  - name: "Testing ansible playbook `deploy-matterbridge.yml`"
-    env: TEST_ROLE=matterbridge
+  - name: "Testing ansible playbook `deploy-update.yml`"
+    env: TEST_ROLE=update
```
Comment (anadahz) [inline .travis.yml:12]: Note, I intentionally test only this role as the other role tests are broken in master.

I have fixed them in: https://github.com/PrivacyLx/devops/pull/37

Comment (francisco-core) [reply .travis.yml:12]: I see. So we merge those first

**Review by francisco-core — APPROVED** [review]
Checked the code and it all looks very sane to me. The review was limited to inspecting the code. I haven't done any actual deployment, but if you've done the testing, I think this can have green light.

Thanks for all the work here!

**Comment by anadahz** [comment]
Fixes: https://github.com/PrivacyLx/privacylx-issue-tracker/issues/64
Closes: https://github.com/PrivacyLx/privacylx-issue-tracker/issues/72

**Review by francisco-core — APPROVED** [review]
As soon as it passes the automatic tests I think we can merge it.

**Comment by anadahz** [comment]
Added some minor linting fixes in: https://github.com/PrivacyLx/devops/pull/50/commits/105a5e7f9398bf0e16c08cba66c18a77c199ca02

CI tests are OK: https://travis-ci.org/github/PrivacyLx/devops/jobs/756470317

**Pull request merged**